app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt

```diff
@@ -230,7 +230,8 @@ class PatientsTabScreen : BaseScreen<
     homeIllustration.setImageResource(illustrationResourceId())
     simpleVideoIllustration.setImageResource(videoIllustrationResourceId())
 
-    if (features.isEnabled(MonthlyDrugStockReportReminder)) {
+    val isMonthlyDrugStockReminderEnabledInIndia = features.isEnabled(MonthlyDrugStockReportReminder) && country.isoCountryCode == Country.INDIA
+    if (isMonthlyDrugStockReminderEnabledInIndia) {
```

Comment:
This prevents us from scheduling drug stock check outside of India. But what about the side effect we are loading when patients screen is created? Should we have a check for that as well?